ui/css/intermittent-failures.css

```diff
@@ -34,6 +34,10 @@ h1,
   display: initial;
 }
 
+.platform-column, .test-suite-column {
+  text-align: left;
```

Comment:
You should be able to use a bootstrap utility instead of creating a new class: https://getbootstrap.com/docs/4.0/utilities/text/

Reply:
similar class names were used in tables. that's why I did this e.g. 

https://github.com/mozilla/treeherder/pull/5639/files#diff-6645f2275cdb79bec9f35b69efb099a5R24

I will use bootstrap class to align.

Reply:
Yes, that should probably be replaced also.

Reply:
updated. I kept the previous class I am not sure if it's acceptable let me know and I will make the changes as necessary

Reply:
Thanks for the change. On second thought, I think the text for all columns in the bug details table should be left aligned (it looks a bit strange to have some columns left aligned and some center aligned).